module/buffer.go

```diff
@@ -19,7 +19,7 @@ type PendingBlockBuffer interface {
 
 	DropForParent(parentID flow.Identifier)
 
-	PruneByHeight(height uint64)
+	PruneByView(view uint64)
```

Comment:
brief goDoc would be great specifically stating whether `view` is included in pruning or not:
```suggestion
	// PruneByView prunes any pending blocks with views less or equal to the given view.
	PruneByView(view uint64)
```